src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
     private String password;
```

Comment:
It's up to you, but also you can add some annotation for the password)

Reply:
I did all password validation in a PasswordValidator class, so there is no need for extra annotations